doc/releases/release-notes-3.7.rst

```diff
@@ -846,6 +853,15 @@ Drivers and Sensors
 
   * The update channels function is now optional and can be left unimplemented.
 
+  * The ``in-gpios`` and ``output-pin`` properties of the respective
+    :dtcompatible:`worldsemi,ws2812-gpio` and :dtcompatible:`worldsemi,ws2812-rpi_pico-pio`
+    devicetree bindings have been renamed to ``gpios``.
+
+  * Removed :kconfig:option:`WS2812_STRIP` and :kconfig:option:`WS2812_STRIP_DRIVER` Kconfig
```

Comment:
```suggestion
  * Removed :kconfig:option:`CONFIG_WS2812_STRIP` and :kconfig:option:`CONFIG_WS2812_STRIP_DRIVER` Kconfig
```
I missed this one before

Reply:
.. ok this is a bit silly.. As these kconfigs are removed you cannot use `:kconfig:option:`
Best just refer to them as ``CONFIG_WS2812_STRIP`` and ``CONFIG_WS2812_STRIP_DRIVER``

Reply:
2 broken links in the release notes

Reply:
As you pointed out @aescolar those would have always been broken links as the options have been removed anyways :)
I will try to find time today to line up a "final cleanup" PR to address all outstanding typos, formatting issues etc